x/migration/module/cmd/claim_morse_application.go

```diff
@@ -75,8 +75,9 @@ func runClaimApplication(cmd *cobra.Command, args []string) error {
 
 	// Retrieve and validate the service ID based on the second argument provided.
 	serviceID := args[1]
-	if !sharedtypes.IsValidServiceId(serviceID) {
-		return ErrInvalidUsage.Wrapf("invalid service ID: %q", serviceID)
+	err = sharedtypes.IsValidServiceId(serviceID)
+	if err != nil {
```

Comment:
```suggestion
	if err := sharedtypes.IsValidServiceId(serviceID); err != nil {
```
Any reason to not have a single liner here?